src/benchmark/file_io_read_micro_benchmark.cpp

```diff
@@ -67,16 +115,15 @@ void read_data_using_libaio(const size_t thread_from, const size_t thread_to, in
   memset(&ctx, 0, sizeof(ctx));
   io_setup(REQUEST_COUNT, &ctx);
 
-  auto batch_size_thread =
-      static_cast<uint64_t>(std::ceil(static_cast<float>(NUMBER_OF_ELEMENTS_PER_THREAD) / REQUEST_COUNT));
+  const auto batch_size_thread = static_cast<uint64_t>(NUMBER_OF_ELEMENTS_PER_THREAD / REQUEST_COUNT);
 
   auto iocbs = std::vector<iocb>(REQUEST_COUNT);
   auto iocb_list = std::vector<iocb*>(REQUEST_COUNT);
 
   for (auto index = size_t{0}; index < REQUEST_COUNT; ++index) {
     auto from = batch_size_thread * index + thread_from;
     auto to = from + batch_size_thread;
-    if (to >= NUMBER_OF_ELEMENTS_PER_THREAD) {
+    if (to + batch_size_thread >= NUMBER_OF_ELEMENTS_PER_THREAD) {
       to = NUMBER_OF_ELEMENTS_PER_THREAD;
     }
```

Comment:
What does `to` do?
Is it accessed somewhere?

Reply:
Removed.